tests/test_iteration_6_features.py

```
@@ -20,22 +20,32 @@
+            pass
+        # Verify credentials actually work — avoid false-positive _LIVE=True
+        # when backend is up but credentials don't match.
+        response = requests.post(
+            f"{url}/api/auth/login",
+            timeout=2,
+            json={"email": ADMIN_EMAIL, "password": ADMIN_PASSWORD},
+        )
+        return response.status_code == 200
     except OSError:
         return False
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)